<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite name="reports.ExtentReportDemo" tests="2" time="0.005" ignored="0" failures="0" timestamp="24 Aug 2021 10:33:30 GMT" skipped="1" hostname="LP-5CD73107R8" errors="1">
  <testcase name="login" time="0.005" classname="reports.ExtentReportDemo">
    <skipped/>
  </testcase> <!-- login -->
  <testcase name="config" time="0.000" classname="reports.ExtentReportDemo">
    <error type="java.lang.NoSuchFieldError" message="VERSION_2_3_29">
      <![CDATA[java.lang.NoSuchFieldError: VERSION_2_3_29
	at com.aventstack.extentreports.reporter.BasicFileReporter.start(BasicFileReporter.java:98)
	at com.aventstack.extentreports.ReportObservable.register(ReportObservable.java:192)
	at java.util.Spliterators$ArraySpliterator.forEachRemaining(Unknown Source)
	at java.util.stream.ReferencePipeline$Head.forEach(Unknown Source)
	at com.aventstack.extentreports.ExtentReports.attachReporter(ExtentReports.java:82)
	at reports.ExtentReportDemo.config(ExtentReportDemo.java:31)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:515)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:217)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:144)
	at org.testng.TestRunner.beforeRun(TestRunner.java:634)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:152)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:57)
]]>
    </error>
  </testcase> <!-- config -->
</testsuite> <!-- reports.ExtentReportDemo -->
